docs/remote/containers.md

```diff
@@ -278,27 +278,36 @@ When you rebuild and reopen in your container, the features you selected will be
 
 You'll get IntelliSense when editing the `"features"` property in the `devcontainer.json` directly:
 
-![Intellisense when modifying terraform feature](images/containers/features-intellisense.png)
+![Intellisense when modifying terraform Feature](images/containers/features-intellisense.png)
 
-Built-in features are sourced from the [script library](https://github.com/microsoft/vscode-dev-containers/tree/main/script-library/docs) folder in the vscode-dev-containers repo, but the Remote - Containers extension and GitHub Codespaces include an **early preview** for creating your own dev container features. For example, you can reference these as follows:
+The **Remote-Containers: Configure Container Features** command allows you to update an existing configuration.
+
+The Features sourced in VS Code UI come from the [`devcontainers/features` repository](https://github.com/devcontainers/features).
+
+### Folder structure
```

Comment:
Maybe "Creating your own feature" as a heading? Then the next sentence could say "It's also easy to create and publish your own Dev Container Features. Published features are stored and shared as [OCI Artifacts](https://github.com/opencontainers/artifacts) in any container registry."

Once Josh merges Once https://github.com/devcontainers/devcontainers.github.io/pull/27, we could point to this for a list of currently published features.

Reply:
Updated the heading and added both the sentence you mention and a reference to the currently published list. Let me know what you think.

Reply:
LGTM!